commitizen/changelog.py

```diff
@@ -130,14 +130,21 @@ def generate_tree_from_commits(
 
 
 def order_changelog_tree(tree: Iterable, change_type_order: List[str]) -> Iterable:
+    if len(set(change_type_order)) != len(change_type_order):
+        raise RuntimeError(
```

Comment:
We prefer to make all the known exceptions as `CommitizenException` (https://github.com/commitizen-tools/commitizen/blob/master/commitizen/exceptions.py#L28). Could you please create one, add an exit code and update the documentation here? https://github.com/commitizen-tools/commitizen/blob/master/docs/exit_codes.md

Reply:
Okay! I added a general `InvalidConfigurationError` for configuration errors like this with a new exit code (I will also add a custom exception to `cz_legacy` as well)